src/components/HeaderBar/index.tsx

```diff
@@ -1,27 +1,30 @@
 import React from 'react'
 import styles from "./HeaderBar.module.scss"
-import Image from 'next/image'
-import AuthComponent from "src/components/Auth"; 
-import logo from "public/tklogo.png"
-import menu from "public/menu-icon.svg"
+import { GiHamburgerMenu } from "react-icons/gi";
+import MobileNavMenu from 'src/components/MobileNavMenu';
+import Link from 'next/link';
 
-class HeaderBar extends React.Component {
-  render() {
-    return(
-      <div className = {styles.Content}>
-        {/* <AuthComponent/> */}
-        <Image src={logo} alt="logo" width={80} height={35} />
-        
-        <div className = {styles.SearchContainer}>
-          
+
+const HeaderBar: React.FC = () => { 
+  const [open, setOpen] = React.useState(false);
+  return(
+    <div className={styles.container}>
+      <div className = {styles.content}>
+        <img src="/tklogo.png" className={styles.logo} />
+        <div className={styles.desktopLinks}>
+          <Link href="#">Tree Map</Link>
+          <Link href="#">Tree Statistics</Link>
+          <Link href="#">Want to Adopt?</Link>
+        </div> 
+        <div className = {styles.searchContainer}> 
           <input type="text" alt="search box" placeholder="Search here" className={styles.SearchBox}/>
-          <div className={styles.Profile} />
-          
+          <div className={styles.profile} /> 
```

Comment:
Stupid question, why do we have a green dot there? Is it a 'search' button?

Reply:
That green dot is a placeholder for this:
![Screen Shot 2022-03-02 at 3 52 06 PM](https://user-images.githubusercontent.com/25257772/156447732-97860741-8a33-4a02-a201-6876c7ea829f.png)
I think it's supposed to display when a user is logged in

Reply:
Ah, gotcha!